### PR TITLE
Update マニュアル.md

### PR DESCRIPTION
**Pull request by andam0202**
Translated the current README.md so that native Japanese speakers can easily understand it.

- Translated only the manual.md directly under the repository, not the manual.md in the doc folder.
- Titles of games sold in Japan have been changed to their Japanese names.
- The links to the games were not changed to the Japanese sites, but left as they are.

This is my first pull request, so sorry if I'm wrong.
Ask me questions if you still find something wrong.

Sounds like an interesting project!
I'm willing to help out a little bit.
If there are any major changes in the README.md I would like to translate it myself.

### CODE REVIEW

**Comment by Sauceke** [comment]
I probably won't be able to keep this file up to date in the long run, but this looks great!

File: マニュアル.md [hunk 1]
```diff
@@ -1,51 +1,203 @@
...
+- [Lovense Calor]
+- The Xbox gamepad
+
+回転機能付きオナホ
+- Vorze A10 Cyclone
+
+[LoveMachine.Experiments]プラグインは、[コイカツ！]と[コイカツ！サンシャイン]で実験的な2つの深度センシングデバイス[Lovense Calor]と[Hotdog]のサポートを追加します。
+
+## インストール
+Download and run the [installer]. If you encounter the "Windows protected your PC" message, click More info > Run anyway.
```
Comment (Sauceke) [inline マニュアル.md:56]: Do you need help in translating this part?
I don't have access to a Windows machine right now, but this is what the message looks like in English: https://i.stack.imgur.com/3t0tg.jpg

Comment (Sauceke) [inline マニュアル.md]: This line is no longer needed, as you have translated it below.

File: マニュアル.md [hunk 2]
```diff
@@ -1,51 +1,203 @@
...
+
+## 使い方
+1. Intiface Centralを起動します。
+2. プレイボタンをクリックしてください。
+3. 使いたいデバイスを起動して接続してください。
+4. ゲーム開始！
+
+スペースキーは、ゲーム中にすべてのデバイスを停止する停止ボタンとして機能します。デバイスを再起動するにはF8キーを推してください。これらのキー割り当ては、プラグイン設定 > LoveMachine > Kill Switch Settingsで変更できます。
+
+⚠ In certain games, the kill switch may not work if the BepInEx console is open while playing in VR, because it can steal focus from the game window. It is recommended to disable the console.
```
Comment (Sauceke) [inline マニュアル.md]: This line is no longer needed, as you have translated it below.

**Review by Sauceke — CHANGES_REQUESTED** [review]

File: マニュアル.md [hunk 3]
```diff
@@ -1,51 +1,203 @@
...
 * Xboxゲームパッド（振動）
 * [そしてこのリストにあるデバイスほぼすべて](https://iostindex.com/?filter0ButtplugSupport=4)
 
+## ご協力してくれる方へ
+対応してほしいゲームタイトルがあれば是非ご連絡ください。新しいゲームタイトルに対応するのは比較的簡単で、ほとんどコーディングをしなくてもよいのです。参考としてPlayHomeの実装をご覧ください。また、対応してほしいデバイスのPRも大歓迎です。
 
-## 使い方
+このMODは無料ですが、寄付を受け付けています。私の開発を応援してくださる方は、[こちら]からよろしくお願いします。
+### 開発者
+Sauceke   •   nhydock   •   hogefugamoga   •   RPKU
```
Comment (Sauceke) [inline マニュアル.md]: You are contributing to the code base, so your name should go here.

**Comment by Sauceke** [comment]
This file is the "live" (deployed) version of the readme, which is overwritten with ``doc/マニュアル.md`` on each release.
Please also overwrite ``doc/マニュアル.md`` with an identical copy of this file.

File: マニュアル.md (hunk 3 above)
Comment (andam0202) [reply マニュアル.md]: Thank you very much. I am afraid I will add my name there.

File: マニュアル.md (hunk 1 above)
Comment (andam0202) [reply マニュアル.md]: The relevant section has been deleted.

File: マニュアル.md (hunk 2 above)
Comment (andam0202) [reply マニュアル.md]: The relevant section has been deleted.

File: doc/マニュアル.md [hunk 4]
```diff
@@ -1,51 +1,196 @@
...
-* **セクロスフィア**
-* **Last Evil**
-* **サキュバス カフェ**
-
-LoveMachineは[buttplug](https://buttplug.io/)というオープンソースプロトコルのお陰でいろんなデバイスと繋がることが出来ます。
-
-現在互換性があるデバイスは：
+LoveMachineはアダルトゲームのキャラクターと連動してアダルトグッズを動かすプラグインです。 
+
+[一部の自律型デバイス](#supported-devices)について、以下のゲームに対応しています。
```
Comment (Sauceke) [inline doc/マニュアル.md]: Broken link
```suggestion
[一部の自律型デバイス](#サポートしているデバイス)について、以下のゲームに対応しています。

```

File: マニュアル.md [hunk 5]
```diff
@@ -1,51 +1,196 @@
...
-* **セクロスフィア**
-* **Last Evil**
-* **サキュバス カフェ**
-
-LoveMachineは[buttplug](https://buttplug.io/)というオープンソースプロトコルのお陰でいろんなデバイスと繋がることが出来ます。
-
-現在互換性があるデバイスは：
+LoveMachineはアダルトゲームのキャラクターと連動してアダルトグッズを動かすプラグインです。 
+
+[一部の自律型デバイス](#supported-devices)について、以下のゲームに対応しています。
```
Comment (Sauceke) [inline マニュアル.md:11]: Broken link
```suggestion
[一部の自律型デバイス](#サポートしているデバイス)について、以下のゲームに対応しています。

```

**Review by Sauceke — CHANGES_REQUESTED** [review]

**Review by Sauceke — APPROVED** [review]

**Comment by Sauceke** [comment]
I haven't merged it yet ^^

**Comment by Sauceke** [comment]
Thank you, your changes have now been deployed.

**Pull request merged**